packages/mdc-textfield/icon/README.md

```diff
@@ -0,0 +1,76 @@
+<!--docs:
+title: "Text Field Icon"
+layout: detail
+section: components
+excerpt: "Icons describe the type of input a text field requires"
+iconId: text_field
+path: /catalog/input-controls/text-fields/icon/
+-->
+
+# Text Field Icon
+
+Icons describe the type of input a text field requires. They can also be interaction targets.
+
+## Design & API Documentation
```

Comment:
See comment in top-level README; I think the templating aspects should be left there and just document the foundation/adapter here.